cherrypy/wsgiserver/ssl_builtin.py

```diff
@@ -37,23 +37,28 @@ class BuiltinSSLAdapter(wsgiserver.SSLAdapter):
     certificate_chain = None
     """The filename of the certificate chain file."""
 
+    ciphers = None
+    """The ciphers list of SSL"""
     """The ssl.SSLContext that will be used to wrap sockets where available
     (on Python > 2.7.9 / 3.3)
     """
     context = None
 
-    def __init__(self, certificate, private_key, certificate_chain=None):
+    def __init__(self, certificate, private_key, certificate_chain=None, ciphers=None):
         if ssl is None:
             raise ImportError('You must install the ssl module to use HTTPS.')
         self.certificate = certificate
         self.private_key = private_key
+        self.ciphers = ciphers
         self.certificate_chain = certificate_chain
         if hasattr(ssl, 'create_default_context'):
             self.context = ssl.create_default_context(
                 purpose=ssl.Purpose.CLIENT_AUTH,
                 cafile=certificate_chain
             )
             self.context.load_cert_chain(certificate, private_key)
+            if(self.ciphers != None):
```

Comment:
1) This violates the code style: you should have space after `if` and parentheses are normally omitted.
2) In Python checking against `None` would be done with `self.ciphers is None`.